src/api/app/models/repository.rb

```diff
@@ -150,6 +150,7 @@ def cycles(arch)
   # or empty list
   def linking_repositories
     return [] if links.size.zero?
+    # FIXME: This is the same as using a `has_many through:` association
```

Comment:
[nitpick] not exactly... this returns an array...  but yes... it shouldn't be like that

Reply:
It's mostly to help us in the future (to hopefully refactor this)

Reply:
That is fine, but the comment could be more accurate :rofl: it is just a nitpick

Reply:
Please just change this. We only use this at [one](https://github.com/openSUSE/open-build-service/search?q=linking_repositories&unscoped_q=linking_repositories). So this shouldn't take much time.